docs/devops.rst

```diff
@@ -0,0 +1,35 @@
+Devops
+===========
+
+Azure Pipeline
```

Comment:
```suggestion
Continuous Integration
```

Reply:
I put Azure pipeline because I did not want to also write the documentation for the other CI we are using, but I was expecting that at one point we should. So I'll put Continuous integration as section, and `Azure DevOps` as sub-section.